resources/migrations/000_migrations.yaml

```diff
@@ -8472,6 +8472,62 @@ databaseChangeLog:
                                   'DateTimeField', 'DecimalField', 'DictionaryField', 'FloatField', 'IntegerField',
                                   'TextField', 'TimeField', 'UUIDField', 'UnknownField');
 
+  - changeSet:
+      id: 307
+      author: tsmacdonald
+      comment: Added 0.40.0
+      changes:
+        - createTable:
+            tableName: comment
+            remarks: "Comments (used for moderation requests and reviews in BUCM)"
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "most recent modification time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
+                  remarks: "creation time"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: text
+                  type: text
+                  remarks: "Contents of the comment"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: commented_item_id
+                  type: int
+                  remarks: "The item being commented on"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: commented_item_type
+                  type: varchar(255)
+                  remarks: "See enum in model file"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author_id
+                  type: int
+                  remarks: "ID of the user who made the comment"
+                  constraints:
+                    nullable: false
```

Comment:
sorry I'm poking around early to see what the interface looks like 🙂 . The FE needs a way to map comments to the specific moderation request being commented on, so won't we need a mod request id on comments as well?

Reply:
that makes comments a little less general unfortunately, so maybe there's a way around that

Reply:
Sorry, I got my wires crossed between writing the schema and writing the rest of it. I think the schema is sound, but the `commented_item_type` should be a _moderation request/review_, not a card/dashboard.

author_id is to show, i.e., Paul's name in this:

![image](https://user-images.githubusercontent.com/784417/119059556-2c09ba00-b996-11eb-9b3a-633f2186232e.png)